master_server.py

```diff
@@ -47,13 +54,21 @@ def main(args):
     """Master server main function."""
     servers, has_ui = create_servers(silent=args.silent,
                                      debug_mode=args.debug_mode)
+    broadcast_queues = {"FmpServer": queue.Queue(), "OpnServer": queue.Queue(),
+                        "LmpServer": queue.Queue(), "RfpServer": queue.Queue()}
     threads = [
-        threading.Thread(target=server.serve_forever)
+        threading.Thread(target=server.serve_forever, name=server.__class__.__name__,
+            args=(broadcast_queues[server.__class__.__name__], ))
         for server in servers
     ]
     for thread in threads:
         thread.start()
 
+    def broadcast_message(text):
+        broadcast = FMP.construct_broadcast(text)
+        for broadcast_segment in broadcast:
+            broadcast_queues["FmpServer"].put(broadcast_segment, block=True)
+
```

Comment:
Maybe you should add a comment stating that it's for easier use in interactive mode. Moreover, you shouldn't use `"FmpServer"` string and prefer using `.__class__.__name__` when possible.

Reply:
Done